### PR TITLE
[BEAM-338] Cleanup Spark runner test resources to avoid notice update

### PR DESCRIPTION
**Pull request by jbonofre**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [X] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [X] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [X] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [X] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---
Cleanup Spark runner test resources to avoid NOTICE & LICENSE update.

### CODE REVIEW

**Comment by jbonofre** [comment]
R: @amitsela

File: runners/spark/src/test/resources/person.avsc [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
- * License.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
  */
 {
-"namespace": "com.cloudera.dataflow.spark.test",
+"namespace": "org.apache.beam.spark.test",
```
Comment (amitsela) [inline runners/spark/src/test/resources/person.avsc]: Should this be `org.apache.beam.runners.spark.test` instead ? just to play nicely with the runner's namespace I guess.

**Comment by amitsela** [comment]
Besides my not about the namespace name, LGTM.

File: runners/spark/src/test/resources/person.avsc (hunk 1 above)
Comment (jbonofre) [inline runners/spark/src/test/resources/person.avsc]: Good point, let me fix that.

**Pull request closed without merging**